packages/cacti-plugin-consortium-static/src/main/json/openapi.json

```diff
@@ -0,0 +1,498 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Hyperledger Cactus Plugin - Consortium Web Service",
+    "description": "Manage a Cactus consortium through the APIs. Needs administrative privileges.",
+    "version": "2.0.0-rc.2",
+    "license": {
+      "name": "Apache-2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "components": {
+    "schemas": {
+      "Identity":{
+        "type": "object",
+        "required": ["pubKey", "memberId", "proof"],
+        "properties": {
+          "pubKey":{
+            "description": "TODO",
```

Comment:
Please add a simple description to the properties of the new schemas. If schemas are already existing feel free to copy its description

Reply:
@RafaelAPB @eduv09 I would also like to spread the word about cross-spec references now being "easy" compared to how broken and complicated they were before (back when we used the HTTP URLs directly pointing at GitHub).

I don't want to derail this PR so this is not a change request, I'm just saying it  for future reference that since the introduction of the `openapi.tpl.json` mechanism you can directly reference other openapi.json files through relative paths on the local file-system ACROSS different packages. If you do a text search in the project for `"$ref": "../` then you can see examples of how it can be used (see screenshot below)

With this new mechanism we could move a lot of the centrally used types to the core-api package and all the other packages could re-use them from there instead of the endless copy-pasting (which then also leads to annoying bugs in the build and even at runtime occasionally).

Again, I'm not saying to change it right now, just something to keep in mind going forward as a helpful alternative for newer things or for refactors later as you see fit.

I also post this comment on the chat and the mailing list to make sure people are aware of this new technique we've enabled recently. 

![image](https://github.com/user-attachments/assets/b58d22de-47c1-4388-b4a7-0906726c0198)

Reply:
This looks great, thanks for sharing Peter. Some of these types are being re-declared consistently, and this fixes that!